jinahub/indexers/storage/MongoDBStorage/mongohandler.py

```diff
@@ -0,0 +1,102 @@
+__copyright__ = "Copyright (c) 2021 Jina AI Limited. All rights reserved."
+__license__ = "Apache-2.0"
+
+from typing import Optional
+
+import numpy as np
+from pymongo import MongoClient
+from jina.logging.logger import JinaLogger
+from jina import Document, DocumentArray
+
+
+def doc_without_embedding(d: Document):
+    new_doc = Document(d, copy=True)
+    new_doc.ClearField('embedding')
+    new_doc.ClearField('id')
+    return new_doc.dict()
```

Comment:
I think this needs to be `SerializeToString`? Can you add a test and make sure the contents of the dump are the same as in the PSQL / LMDB?

Reply:
okay